coffee-module/coffee-module-mp/coffee-module-mp-restclient/src/main/java/hu/icellmobilsoft/coffee/module/mp/restclient/provider/DefaultLoggerClientRequestFilter.java

```diff
@@ -49,7 +54,7 @@
  */
 @Priority(value = RestClientPriority.REQUEST_LOG)
 @Dependent
-public class DefaultLoggerClientRequestFilter implements ClientRequestFilter {
+public class DefaultLoggerClientRequestFilter implements ClientRequestFilter, WriterInterceptor {
```

Comment:
ClientRequestFilter igy mar nem is kellene, mindent ami loggolunk a WriterInterceptor-bol kellene kiszedni.

Reply:
cookiet és url-t nem tudom logolni vele, nem érem a WriterInterceptorContext-ből.
Esetleg lehet megoldás hogy mivel egymás után vannak összekötni a 2 logot requestScope-al, es az interceptorban logolni egyben, csak akkor reactivan nem fog működni később

Reply:
Most igy 2 log sor fog keletkezni, amit 2 filter/interceptor kulon keszit. Esetleg valami CDI.select, context valtozoba vagy akkor a requestscope valtozoba be kellene gyujteni a resz informaciokat es egyszerre egy sorban printelni